tests/unit/elements/UserElementTest.php

```diff
@@ -81,6 +81,27 @@ public function testValidateUnverifiedEmail(): void
         $this->tester->deleteElement($user);
     }
 
+    public function testUniqueAttributesForInactiveUsers(): void
+    {
+        $user = new User([
+            'active' => true,
+            'email' => $this->inactiveUser->email,
+            'username' => $this->inactiveUser->username,
+            'unverifiedEmail' => $this->inactiveUser->unverifiedEmail,
+        ]);
+
+        $this->tester->saveElement($user);
+
+        self::assertFalse($user->hasErrors());
```

Comment:
User is able to be saved, even though an inactive user has the same email/username.